### PR TITLE
Market maker update all unit tests

### PR DESCRIPTION
**Pull request by kvbik**
json objects instead of comments and couple helper functions

### CODE REVIEW

**Comment by kvbik** [comment]
need to make the javascript work, but feel free to check the numbers are ok

File: test/MarketMaker.js [hunk 1]
```diff
@@ -145,146 +145,168 @@ contract('marketMaker', function(accounts) {
...
+    // MMETH = 38,    TP = 68,   MMT = 100, TC = 0,   SP = 2163 mETH/Lif, BP = 800,  CL 60%, maxClaimable = 0,   claimed = 30
+    checkScenarioProperties({
+      marketMakerEthBalance: 38, marketMakerLifBalance: 100, totalProfit: 68,
+      customerEthBalance: 632, customerLifBalance: 0,
+      marketMakerSellPrice: 2163, marketMakerBuyPrice: 800,
+      claimablePercentage: 60, maxClaimableEth: 0, totalClaimedEth: 30
+    });
+
+    // Buy 100 tokens
+    await token.approve(customer, 100);
```
Comment (nberger) [inline test/MarketMaker.js]: no need to approve when buying tokens from the market maker

File: test/MarketMaker.js [hunk 2]
```diff
@@ -145,146 +145,168 @@ contract('marketMaker', function(accounts) {
...
-  // Claimable month: 366
-  // Claimable total: 19+154+366 = 543
-  // Claim(243);
-  // Claimable total: 300
-  // claimableUpdatedMonth: 3
-
-  // Month 4:
-  // Claimable month: 651
-  // Claimable total: 300+651 = 951
+  var checkScenarioProperties = async function(data, mm, customer) {
```
Comment (AugustoL) [inline test/MarketMaker.js]: periodIndex is missing here

Comment (kvbik) [inline test/MarketMaker.js]: there is a helper function `waitForMonth` called before in the test

Comment (AugustoL) [inline test/MarketMaker.js]: yes, but we want to check that we are in the correct period after we wait, right?

Comment (kvbik) [inline test/MarketMaker.js]: that means we do not believe the helper? and we are checking it all over again. not sure...

**Pull request merged**